client/documents/index.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Page from 'components/page';
+import DocumentsList from './list';
+import { TestModeNotice, topics } from 'components/test-mode-notice';
+import {
+	EmptyStateList,
+	EmptyStateTableHeaders,
+} from '../empty-state-table/list';
+import EmptyStateTable from 'empty-state-table';
+// TODO: Need a documents banner created.
+import ListBanner from '../empty-state-table/transactions-banner.svg';
+import { Experiment } from '@woocommerce/explat';
+
+export const DocumentsPage = (): JSX.Element => {
+	const defaultExperience = (
+		<>
+			<TestModeNotice topic={ topics.documents } />
+			<DocumentsList />
+		</>
+	);
+
+	const treatmentExperience = wcpaySettings.accountStatus.status ? (
+		defaultExperience
+	) : (
+		<EmptyStateTable
+			headers={ EmptyStateTableHeaders }
+			title="Documents"
+			content={ <EmptyStateList listBanner={ ListBanner } /> }
+		/>
+	);
+
+	return (
+		<Page>
+			<Experiment
```

Comment:
Should we include this page in this experiment? It doesn't look like the disputes page is included.

Reply:
Like the banner, I've removed the code referencing the experiment in https://github.com/Automattic/woocommerce-payments/pull/4000/commits/04bfa6ae854ccf8c8cc5621085a85418f094735a